rpm/qt6-amber-mpris.spec

```diff
@@ -0,0 +1,77 @@
+Name:       qt6-amber-mpris
```

Comment:
We've so far had -qt5 postfix on naming on many places. Perhaps this should follow the pattern. And even without thinking about consistency it could be nicer having component name first not to emphasize qt version over it.